calc/src/calc_fee.rs

```diff
@@ -49,6 +49,12 @@ impl Multiplier {
             ("westend", v) if 10 < v && v < 31 => V1((new_i128(inner), false)),
             ("westend", v) if 31 <= v => V2(new_u128(inner)),
 
+            ("statemine", _v) => V2(new_u128(inner)),
```

Comment:
One idea here is to just assume all system parachains have the same fee calc as the current runtime of the relay chain .. not sure if this is a dangerous assumption. But if we are willing to make the assumption we could eliminate the new match logic and plug in the relay chain name and runtime version

Reply:
I just asked DQ's, but I am not totally sure. I would side on the case that making the assumption can add potential work in the future but I am not sure if it is necessarily dangerous because we know the Multiplier it needs to use, and if it changes we'll have to update the code no matter what. I do think though that explicitly stating the `Multiplier` directly to the specName (in this case `statemine`) is good for clarity and readability.

Reply:
Okay so based on this response:

> System parachains may be optimized for specific operations and could have different fee structures as a result. Also, unlike the relay chain, PoV size factors into the weight of a parachain block, which causes a fundamental difference between the relay chain and parachain fee structures

I would say it's probably best we keep it as is, also important for me to keep an eye on potential changes that could happen to the fee structure.

Reply:
Ok, let's play it safe then. :)